### PR TITLE
:bug: Source Google Directory: migrate to advanced auth

### PR DESCRIPTION
**Pull request by davydov-d**
## What
#26232

## How
Remove `authSpecification` from the connector spec because the OAuth flow is not yet supported for this connector in cloud env

### CODE REVIEW

File: airbyte-integrations/connectors/source-google-directory/source_google_directory/spec.json [hunk 1]
```diff
@@ -78,13 +78,5 @@
         ]
       }
     }
-  },
-  "authSpecification": {
```
Comment (girarda) [inline airbyte-integrations/connectors/source-google-directory/source_google_directory/spec.json:82]: do we automatically convert authSpecification to advancedAuth or is this a breaking change?

Comment (davydov-d) [reply airbyte-integrations/connectors/source-google-directory/source_google_directory/spec.json:82]: @girarda this is not a breaking change. Both advancedAuth and authSpecification are nullable, besides this connector is outdated on cloud and has no oauth support so we will not break things

**Review by girarda — APPROVED** [review]

**Review by bazarnov — APPROVED** [review]

**Review by lazebnyi — APPROVED** [review]

**Pull request merged**